src/test/java/persistence/sql/ddl/definition/PrimaryKeyQueryableTest.java

```diff
@@ -0,0 +1,44 @@
+package persistence.sql.ddl.definition;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import persistence.sql.ddl.fixtures.EntityWithAutoIdStrategy;
+import persistence.sql.ddl.fixtures.EntityWithIdentityIdStrategy;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PrimaryKeyQueryableTest {
+
+    @Test
+    @DisplayName("GeneratedValue annotation이 없으면 IDENTITY 전략이 아닌 AUTO 전략으로 생성되어야 한다.")
+    public void shouldCreateAutoIdStrategy() throws Exception {
+        // Given
+        Class<EntityWithAutoIdStrategy> clazz = EntityWithAutoIdStrategy.class;
+        PrimaryKey pk = new PrimaryKey(clazz.getDeclaredField("id"));
+
+        // When
+        StringBuilder query = new StringBuilder();
+        pk.apply(query);
+
+        // Then
+        String expectedQuery = "id BIGINT , ";
+        assertThat(query.toString()).isEqualTo(expectedQuery);
```

Comment:
```suggestion
        assertThat(query.toString()).isEqualTo("id BIGINT , ");
```
expected도 어떻게보면 given에 가깝고, 개인적으로는 expected를 거는것도 좋으나 then절에서 직관적으로 드러나는 것이 더 좋다고 생각해서 하드코딩해서 넣는걸 선호합니다 :)